packages/datadog-plugin-aerospike/src/index.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+const { storage } = require('../../datadog-core')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class AerospikePlugin extends DatabasePlugin {
+  static get id () { return 'aerospike' }
+  static get operation () { return 'command' }
+  static get system () { return 'aerospike' }
+  static get prefix () {
+    return 'tracing:apm:aerospike:command'
+  }
+
```

Comment:
This should also report `peer.service` - it seems like the relevant precursor is `aerospike.namespace`.

All you need to do (see for ex. `packages/datadog-plugin-cassandra-driver/src/index.js`) is define the `peerServicePrecursors` static getter to return `['aerospike.namespace']`, and test with the `withPeerService` fixture.